File: 0901_1000/0949/0949.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, m, k = (int(x) for x in f.read().split())
    for i in range(n - 1):
        m, k = k - m, m
    q.write(f"{m} {k}")

File: 0901_1000/0949/0949.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m, k, a;

	fscanf(f, "%d%d%d", &n, &m, &k);
	while (--n)
		a = m, m = k - m, k = a;
	fprintf(q, "%d %d", m, k);
	return 0;
}
